driver/src/io_backend/ext/xaie_amdair.c

```diff
@@ -0,0 +1,615 @@
+/******************************************************************************
+* Copyright (C) 2023 Xilinx, Inc.  All rights reserved.
+* SPDX-License-Identifier: MIT
+******************************************************************************/
+
+/***************************** Include Files *********************************/
+#ifdef __linux__
+#include <pthread.h>
+#endif
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+
+#include "xaie_helper.h"
+#include "xaie_io.h"
+#include "xaie_io_common.h"
+#include "xaie_io_privilege.h"
+#include "xaie_npi.h"
+
+/****************************** Type Definitions *****************************/
+typedef struct {
+	u64 BaseAddr;
+	u64 NpiBaseAddr;
+	char *AmdairAddress;
+	char *AmdairValue;
+} XAie_AmdairIO;
+
+/************************** Function Definitions *****************************/
+/*****************************************************************************/
+/**
+*
+* This is the memory IO function to free the global IO instance
+*
+* @param	IOInst: IO Instance pointer.
+*
+* @return	None.
+*
+* @note		The global IO instance is a singleton and freed when
+* the reference count reaches a zero.
+*
+*******************************************************************************/
+static AieRC XAie_AmdairIO_Finish(void *IOInst)
+{
+	XAie_AmdairIO *DevInst = (XAie_AmdairIO *)IOInst;
```

Comment:
nit: `Amdair` -> `AmdAir`

Reply:
Addressed in [6de16c1](https://github.com/stephenneuendorffer/aie-rt/pull/4/commits/6de16c1da98d0def29a457547e30e8c1a6f0bb43)